environments/prod/appeal-immigration-asylum-decision-service-gov-uk.yml

```
@@ -11,9 +11,9 @@ txt:
 cname:
   - name: "www"
     ttl: 60
-    record: "hmcts-prod.azurefd.net"
-    shutter: false
-    #record: "hmcts-appeal-immigration-asylum-decision-shutter.azureedge.net"
+    #record: "hmcts-prod.azurefd.net"
+    shutter: true
```

I don't think you want to change this if you're using the record method
```suggestion
    shutter: false
```